### ISSUE TITLE
Convert to ES5?

### ISSUE THREAD

**Issue by Thundros** [yomotsu/meshwalk.js#9]
Hello, @yomotsu ! I just would like to ask if you could convert this library to ES5 as opposed to ES6 so that I may study the code if possible since it is MIT licensed. I would very much like to use this as part of my demo but at the same time, I only need to utilize the library functions in my demo. There are some defaults in the library that I cannot use such as using my own Keys library for moving the character & such.

Thank you so much for all that you do & please continue doing it!

Have a great day! <3

Thundros

**Comment by Thundros** [comment]
Hello?

**Comment by Thundros** [comment]
hello?...

**Comment by yomotsu** [comment]
Sorry for the delay.
Actually, ES5 version is already available.
following one is published automatically using rollup.js.
https://github.com/yomotsu/meshwalk.js/blob/master/dist/meshwalk.js

**Comment by Thundros** [comment]
Hi, yomotsu! :) No worries! I was wondering. Is there a way to remove stuff like this?

[code](function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
	typeof define === 'function' && define.amd ? define(['exports'], factory) :
	(factory((global.MW = {})));
}(this, (function (exports) { 'use strict';[/code]

**Comment by yomotsu** [comment]
Do you really need to remove that?
It just works as ES5.

**Comment by Thundros** [comment]
@yomotsu Yes, I do need to remove it so I can study just the classes & not the mangled mess.

**Comment by Thundros** [comment]
Hello?

**Comment by Thundros** [comment]
hello?

**Comment by Thundros** [comment]
?
